internal/testutils/seed.go

```diff
@@ -0,0 +1,96 @@
+package testutils
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/broadinstitute/sherlock/internal/builds"
+	"github.com/broadinstitute/sherlock/internal/services"
+	"gorm.io/gorm"
+)
+
+// SeedServices is a test utility that will populate a database with a predetermined list of "services"
+// to be used for running integration tests against a real database
+func SeedServices(db *gorm.DB) ([]services.Service, error) {
```

Comment:
Can we keep the seeding functions in their respective pacakages rather than `testutils`. This will cause import cycles if I wanted to use other methods from `package testutils` from an integration test in `package services`